services/drupal/web/themes/epa_theme/epa_theme.libraries.yml

```diff
@@ -18,42 +18,10 @@ global:
     - core/drupal
     - core/drupalSettings
     - core/jquery
-    - epa_theme/font_arabic
-    - epa_theme/font_korean
-    - epa_theme/font_simplified_chinese
-    - epa_theme/font_traditional_chinese
 details:
   version: VERSION
   js:
     js/libraries/details-element-polyfill.js: {}
-font_arabic:
-  css:
-    theme:
-      //fonts.googleapis.com/css2?family=Noto+Sans+Arabic:wght@400;700&display=swap: {
-        preprocess: false,
-        type: external,
-      }
-font_korean:
-  css:
-    theme:
-      //fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@400;700&display=swap: {
-        preprocess: false,
-        type: external,
-      }
-font_simplified_chinese:
-  css:
-    theme:
-      //fonts.googleapis.com/css2?family=Noto+Sans+SC:wght@400;700&display=swap: {
-        preprocess: false,
-        type: external,
-      }
-font_traditional_chinese:
-  css:
-    theme:
-      //fonts.googleapis.com/css2?family=Noto+Sans+TC:wght@400;700&display=swap: {
-        preprocess: false,
-        type: external,
-      }
```

Comment:
@jbwagner Can you add these libraries back? Have them point to new CSS files rather than to the google hosted ones. This will allow us to only load specific languages in the future (which is on EPA’s roadmap.)

Reply:
@dcmouyard added them back. Per our Slack convo, I kept the preloads I added for now until we solve for the future case of only loading exactly what we need.